tests/acceptance/features/apiSharingNgAdditionalShareRole/listGrantsShareRole.feature

```diff
@@ -243,3 +243,439 @@ Feature: ListGrants role
       | permissions-role       |
       | Viewer With ListGrants |
       | Editor With ListGrants |
+
+
+  Scenario Outline: sharer updates shared file roles to ListGrants roles (Personal space)
+    Given the administrator has enabled the permissions role "<new-permissions-role>"
+    And user "Alice" has uploaded file with content "to share" to "textfile1.txt"
+    And user "Alice" has sent the following resource share invitation:
+      | resource        | textfile1.txt      |
+      | space           | Personal           |
+      | sharee          | Brian              |
+      | shareType       | user               |
+      | permissionsRole | <permissions-role> |
+    When user "Alice" updates the last resource share with the following properties using the Graph API:
+      | permissionsRole | <new-permissions-role> |
+      | space           | Personal               |
+      | resource        | textfile1.txt          |
+    Then the HTTP status code should be "200"
+    And the JSON data of the response should match
+      """
+      {
+        "type": "object",
+        "required": ["grantedToV2","id","roles"],
+        "properties": {
+          "grantedToV2": {
+            "type": "object",
+            "required": ["user"],
+            "properties":{
+              "user": {
+                "type": "object",
+                "required": ["displayName","id"],
+                "properties": {
+                  "displayName": {"const": "Brian Murphy"},
+                  "id": {"pattern": "^%user_id_pattern%$"}
+                }
+              }
+            }
+          },
+          "id": {"pattern": "^%permissions_id_pattern%$"},
+          "roles": {
+            "type": "array",
+            "minItems": 1,
+            "maxItems": 1,
+            "items": {"pattern": "^%role_id_pattern%$"}
+          }
+        }
+      }
+      """
+    Examples:
```

Comment:
confirm the role change. check what will be the difference

Reply:
up-coming PR holds scenario without json check there i will confirm role change

Reply:
what do the upcoming scenarios do?

Reply:
check activities and others user with whom resource has been shared

Reply:
I would recommend having them in a single scenario when added